docs/sphinx/source/whatsnew/v0.7.1.rst

```diff
@@ -0,0 +1,11 @@
+.. _whatsnew_0701:
+
+v0.7.1 (MONTH DAY, YEAR)
+---------------------
+
+- Add a faster way to calculate the solar position (`spencer_mc`)
```

Comment:
please move this to the 0.7.0 whats new file https://github.com/pvlib/pvlib-python/blob/master/docs/sphinx/source/whatsnew/v0.7.0.rst